scripts/analysis/amwg_table.py

```diff
@@ -287,39 +290,45 @@ def amwg_table(adf):
         #End of var_list loop
         #--------------------
 
-        # Move RESTOM to top of table
-        #----------------------------
-        if 'RESTOM' in var_list:
+        # Move RESTOM to top of table (if applicable)
+        #--------------------------------------------
+        try:
             table_df = pd.read_csv(output_csv_file)
-            table_df = pd.concat([table_df[table_df['variable'] == 'RESTOM'], table_df]).reset_index(drop = True)
-            table_df = table_df.drop_duplicates()
-            table_df.to_csv(output_csv_file, header=cols, index=False)
-        #End if
+            if 'RESTOM' in table_df['variable'].values:
+                table_df = pd.concat([table_df[table_df['variable'] == 'RESTOM'], table_df]).reset_index(drop = True)
+                table_df = table_df.drop_duplicates()
+                table_df.to_csv(output_csv_file, header=cols, index=False)
 
-        # last step is to add table dataframe to website (if enabled):
-        table_df = pd.read_csv(output_csv_file)
-        adf.add_website_data(table_df, case_name, case_name, plot_type="Tables")
+            # last step is to add table dataframe to website (if enabled):
+            table_df = pd.read_csv(output_csv_file)
```

Comment:
I think `table_df` is already read in above, so I believe this line isn't needed (?).